group_3/script.R

```diff
@@ -0,0 +1,122 @@
+# Group 3
+# Matthew Bodziony, Daniel Choi, Cole Groeneveld
+
+# Path to Git Bash on your computer
+git_bash_path <- "C:/Program Files/Git/git-bash.exe"
+
+# Write your bash commands and run them  
+curl_command <- "curl -L -o ./raw_data/archive.zip https://www.kaggle.com/api/v1/datasets/download/tylermorse/retail-business-sales-20172019"
+system(paste(shQuote(git_bash_path), 
+             "-c", 
+             shQuote(curl_command)), 
+       intern = TRUE)
```

Comment:
this script will fail when `raw_data` isn't tracked by Git and when I pull it I don't have this folder. The best way is to create this folder directly with either a bash command or `dir.create()` in R.